djangocms_googlemap/templates/djangocms_googlemap/default/map.html

```diff
@@ -24,9 +24,38 @@
         {% render_plugin plugin %}
     {% endfor %}
 </div>
+{% comment %}
+prevents a disaligne bug of the images in the controls,
+when the map is redrawn (trigger of the event 'cms-content-refresh'),
+taken from googles examples
+{% endcomment %}
+{% addtoblock "css" %}
```

Comment:
please wrap this with with ``{% if request.toolbar and request.toolbar.edit_mode_active or request.toolbar and request.toolbar.edit_mode %}`` to ensure it only loads when the CMS Toolbar is active, otherwise it also renders on live.